telemetry/src/types.rs

```diff
@@ -0,0 +1,41 @@
+/// Encapsulates the retry period that is calculated as `min_period + 2 ^ min(exponent, max_exponent)`
+pub struct RetryPeriod {
+    /// The minimum period
+    min_period: u64,
+    /// The maximum exponent
+    max_exponent: u8,
+    /// The current exponent
+    exponent: u8,
+}
+
+impl RetryPeriod {
+    pub const DEFAULT_MIN_PERIOD: u64 = 1;
+    pub const DEFAULT_MAX_EXPONENT: u8 = 4;
```

Comment:
I think you can define these default values in `::config` and just call it from this module

Reply:
I disagree. They belong to RetryPeriod.